tutorials/s3-to-hdfs-sync/src/site/conf/cluster-memory-conf.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+
```

Comment:
Single configuration is not suitable for all environments. So, we are trying to capture them separately.

properties.xml would capture mandatory properties for which there cannot be system default.
Other properties which are advanced but can be initialized with meaningful defaults are captured under apconf. These can be used using -apconf from apex cli.